website/snippets/core-versions-table.md

```diff
@@ -1,21 +1,21 @@
 ### Latest Releases
 
-| dbt Core                        | Initial Release | Support Level | Critical Support Until  | dbt Cloud Until |
-|---------------------------------|-----------------|---------------|-------------------------|-----------------|
-| [**v1.4**](upgrading-to-v1.4)   | Jan 25, 2023    | Active        | Jan 25, 2024            | Jan 2024        |
-| [**v1.3**](upgrading-to-v1.3)   | Oct 12, 2022    | Critical      | Oct 12, 2023            | Oct 2023        |
-| [**v1.2**](upgrading-to-v1.2)   | Jul 26, 2022    | Critical      | Jul 26, 2023            | Jul 2023        |
-| [**v1.1**](upgrading-to-v1.1)   | Apr 28, 2022    | Critical      | Apr 28, 2023            | Apr 2023        |
-| [**v1.0**](upgrading-to-v1.0) ⚠️ | Dec 3, 2021     | End of Life ⚠️ | Dec 3, 2022 ⚠️             | Pending deprecation        |
-|  **v0.X** ⚠️                     | (Various dates) | End of Life ⚠️ | Deprecated ⚠️             | Deprecated ⚠️     |
+| dbt Core                                                   | Initial Release | Support Level | Critical Support Until  | dbt Cloud Until |
+|------------------------------------------------------------|-----------------|---------------|-------------------------|-----------------|
+| [**v1.5**](/guides/migration/versions/upgrading-to-v1.5)   | Apr 27, 2023    | Active        | Apr 27, 2024            | Apr 2024        |
+| [**v1.4**](/guides/migration/versions/upgrading-to-v1.4)   | Jan 25, 2023    | Critical      | Jan 25, 2024            | Jan 2024        |
+| [**v1.3**](/guides/migration/versions/upgrading-to-v1.3)   | Oct 12, 2022    | Critical      | Oct 12, 2023            | Oct 2023        |
+| [**v1.2**](/guides/migration/versions/upgrading-to-v1.2)   | Jul 26, 2022    | Critical      | Jul 26, 2023            | Jul 2023        |
+| [**v1.1**](/guides/migration/versions/upgrading-to-v1.1) ⚠️ | Apr 28, 2022    | End of Life ⚠️ | Apr 28, 2023            | Apr 2023        |
+| [**v1.0**](/guides/migration/versions/upgrading-to-v1.0) ⚠️ | Dec 3, 2021     | End of Life ⚠️ | Dec 3, 2022 ⚠️           | Pending deprecation|
+|  **v0.X** ⚠️                                                | (Various dates) | End of Life ⚠️ | Deprecated ⚠️            | Deprecated ⚠️    |
 
 ### Planned future releases
 
 _Future release dates are tentative and subject to change._
 
 | dbt Core | Planned Release | Critical & dbt Cloud Support Until  |
 |----------|-----------------|-------------------------------------|
-| **v1.5** | _Apr 2023_      | _Apr 2024_                          |
 | **v1.6** | _July 2023_     | _July 2024_                         |
 | **v1.7** | _Oct 2023_      | _Oct 2024_                          |
-| **v1.8** | _Jan 2024_      | _Jan 2025_                          |
+| **v1.8** | _Jan 2024_      | _Jan 2025_                          |
```

Comment:
Maybe add v1.9 too so that we're always keeping 12 months ahead of our planned releases?

```suggestion
| **v1.8** | _Jan 2024_      | _Jan 2025_                          |
| **v1.9** | _Apr 2024_      | _Apr 2025_                          |
```

Reply:
i think that's a good idea and helpful to users. adding this to the table @matthewshaver and @dbeatty10 - if it's incorrect, we can remove.